src/components/ec-table-filter/ec-table-filter.story.js

```diff
@@ -0,0 +1,62 @@
+import EcTableFilter from './ec-table-filter.vue';
+import EcSyncMultipleValuesFilter from '../ec-sync-multiple-values-filter';
+import EcDateRangeFilter from '../ec-date-range-filter';
+
+export default {
+  title: 'Filters/Table Filter',
+  component: EcTableFilter,
+};
+
+export const basic = (args, { argTypes }) => ({
```

Comment:
I am not strong opinionated about it and it perfectly works as is, but maybe we can do this,

`const Template = (args, { argTypes }) => ({`

and then at the bottom do 

```
export const basic = Template.bind({});

basic.args = {
  value: { paymentStatus: [{ text: 'Paid', value: 'paid' }, { text: 'Cancelled', value: 'canceled' }], feeType: [{ 
  ...
```

One of the benefits of writing stories the new way is reusable templates. Some stories might not need one but maybe they will do in the future and some others they will need one. To keep it consistent maybe is worth having all in the following way
https://storybook.js.org/docs/react/writing-stories/introduction#using-args

Reply:
I thought it was redundant to use if I wasn't not going to re-use it that's why Im not creating the template ur if you guy think we should always then I'll change it

Reply:
changed it to help with consistency and for future stories we'd even need